code/_onclick/xeno.dm

```diff
@@ -54,6 +54,8 @@
 					var/turf/target_turf = target
 					for(var/obj/flamer_fire/fire in target_turf)
 						firepatted = TRUE
+						if(!(caste.fire_immunity & FIRE_IMMUNITY_NO_DAMAGE) || fire.tied_reagent.fire_penetrating)
+							apply_damage(fire.burnlevel, BURN,fire)
```

Comment:
Make it a prob or like greatly reduce the damage dealt.

Reply:
Fire itself is just `burnlevel*FIRE_DAMAGE_PER_LEVEL` where FIRE_DAMAGE_PER_LEVEL is 3.

Technically it ought to use `check_fire_intensity_resistance()` too but its not really used.